py_configuration_files/codejail.py

```diff
@@ -12,8 +12,8 @@
 CODEJAIL_ENABLED = True
 
 CODE_JAIL = {
-    # These values are coordinated with the Dockerfile (in edx/public-dockerfiles)
-    # and the AppArmor profile (codejail.profile in edx/devstack).
+    # These values are coordinated with the Dockerfile and the AppArmor profile,
+    # both in edx/public-dockerfiles.
```

Comment:
Nit: I would guess people will be more familiar with docker than AppArmor, so restoring the file name. This is a non-blocking suggestion, in case you feel strongly we shouldn't.
```suggestion
    # These values are coordinated with the Dockerfile and the AppArmor profile
    # (codejail.profile), both in edx/public-dockerfiles.
```

Reply:
Sure, I'll work it in.